website/docs/advanced-concepts/granular-access-control/README.md

```diff
@@ -19,61 +19,43 @@ tags={[
-/>
-
-## Groups
-
-A group is a collection of users that allows you to assign roles to multiple users at once, simplifying permission management. Here are some important things to know about groups:
-
-- You can assign multiple roles to a group, and users can belong to multiple groups.
-- To create a new group, navigate to **Admin Settings** > **Access Control** > **Groups**, and click the **Add group** button in the upper-right corner of the screen.
-- To manage groups and the roles assigned to them, go to **Admin Settings** > **Access Control** > **Groups**. This page shows all the groups, and clicking on one displays the users who are members of the group, along with the roles assigned to them.
-
-When you share an application or workspace, you can invite a group and assign the desired role to it. This makes it easy to grant permissions to multiple users at once, and you can modify the group's roles to update the permissions for all its members simultaneously.
-
-
-## Programmatic access control
-
-You can utilize roles and groups assigned to a user to programmatically manage access to various entities such as widgets, datasources, APIs, and queries. 
-
-To enable Programmatic access control, follow the steps below - 
-
-1. On the homepage, go to **Admin Settings > General**.
-2. Under the **Programmatic Access Control** section, select the **Access roles and user groups in code for conditional business logic** checkbox. 
-
-<ZoomImage
-  src="/img/Enable_programmatic-access-control.png" 
-  alt="Programmatic Access Control"
-  caption="Enable Programmatic access control"
-/>
-
-3. To manage access control programmatically, use the relevant properties of the Appsmith user object `appsmith.user.roles` or  `appsmith.user.groups` in JavaScript code to obtain an array of roles or groups assigned to the logged-in user. For more information, refer to the [roles](/reference/appsmith-framework/context-object#roles) and [groups](/reference/appsmith-framework/context-object#groups) properties of the Appsmith user object.
-
-## Further reading
+Granular Access Control (GAC) in Appsmith allows you to specify precise access permissions for every entity within your Appsmith instance. This includes applications, users, workspaces, queries, and more. Here’s a quick overview of how GAC works:
+
+  <ZoomImage
+   src="/img/gac-overview.png" 
+   alt="How Granular Access Control works in Appsmith"
+   caption="How Granular Access Control works in Appsmith"
+   />
+
+## Core components
+
+* **Permissions** - define what actions a user can perform on specific entities. Every entity in Appsmith whether it’s an app, a page, a datasource and environment, or a query can have granular permissions set for actions like create, read, update, and delete.
+* **Roles** - are collections of permissions. Instead of assigning individual permissions to users or groups, you can bundle permissions into roles. This simplifies the management of permissions.
```

Comment:
A role is a set of permissions you can assign to individuals or user groups.  Roles give users and teams of people varying levels of access to Appsmith resources.